package/src/components/Message/MessageSimple/ReactionList/ReactionListBottom.tsx

```diff
@@ -0,0 +1,278 @@
+          type={reaction.type}
+        />
+        <Text style={[styles.reactionCount, { color: black }, countText]}>{reaction.count}</Text>
+      </Animated.View>
+    </Pressable>
+  );
+};
+
+export type ReactionListBottomProps<
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
+> = Partial<
+  Pick<
+    MessageContextValue<StreamChatGenerics>,
+    | 'handleReaction'
+    | 'hasReactions'
+    | 'onLongPress'
+    | 'onPress'
+    | 'onPressIn'
+    | 'preventPress'
+    | 'reactions'
+    | 'showMessageOverlay'
+  >
+> &
+  Partial<Pick<MessagesContextValue<StreamChatGenerics>, 'supportedReactions'>>;
+
+export const ReactionListBottom = <
+  StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics,
+>(
+  props: ReactionListBottomProps<StreamChatGenerics>,
+) => {
+  const {
+    handleReaction: propHandlerReaction,
+    hasReactions: propHasReactions,
+    onLongPress: propOnLongPress,
+    onPress: propOnPress,
+    onPressIn: propOnPressIn,
+    preventPress: propPreventPress,
+    reactions: propReactions,
+    showMessageOverlay: propShowMessageOverlay,
+    supportedReactions: propSupportedReactions,
+  } = props;
+
+  const {
+    handleReaction: contextHandleReaction,
+    hasReactions: contextHasReactions,
+    onLongPress: contextOnLongPress,
+    onPress: contextOnPress,
+    onPressIn: contextOnPressIn,
+    preventPress: contextPreventPress,
+    reactions: contextReactions,
+    showMessageOverlay: contextShowMessageOverlay,
+  } = useMessageContext<StreamChatGenerics>();
+
+  const { supportedReactions: contextSupportedReactions } =
+    useMessagesContext<StreamChatGenerics>();
+
+  const handleReaction = propHandlerReaction || contextHandleReaction;
+  const hasReactions = propHasReactions || contextHasReactions;
+  const onLongPress = propOnLongPress || contextOnLongPress;
+  const onPress = propOnPress || contextOnPress;
+  const onPressIn = propOnPressIn || contextOnPressIn;
+  const preventPress = propPreventPress || contextPreventPress;
+  const reactions = propReactions || contextReactions;
+  const showMessageOverlay = propShowMessageOverlay || contextShowMessageOverlay;
+  const supportedReactions = propSupportedReactions || contextSupportedReactions;
+  const {
+    theme: {
+      messageSimple: {
+        reactionListBottom: { container },
+      },
+    },
+  } = useTheme();
+
+  const supportedReactionTypes = supportedReactions?.map(
+    (supportedReaction) => supportedReaction.type,
+  );
+
+  const hasSupportedReactions = reactions.some((reaction) =>
+    supportedReactionTypes?.includes(reaction.type),
+  );
+
+  if (!hasSupportedReactions || !hasReactions) {
+    return null;
+  }
+
+  return (
+    <View accessibilityLabel='Reaction List Bottom' style={[styles.container, container]}>
+      {reactions.map((reaction, index) => (
+        <ReactionListBottomItem
+          key={index}
+          {...{
+            handleReaction,
```

Comment:
Although this would achieve the same thing, I would rather have these be separate props for the component. The reason is because destructuring is actually doing extra work where it's not needed. Basically having something like:
```suggestion
            handleReaction={handleReaction},
```
is more explicit and we should keep it that way.